Dockerfile

```diff
@@ -25,19 +25,43 @@ RUN yarn install  --network-timeout 1000000
 
 ## Build application
 RUN yarn build
+RUN yarn build-server
 
-FROM registry.access.redhat.com/ubi8/nginx-122:latest
+FROM registry.access.redhat.com/ubi8/nodejs-20-minimal:latest
 
-USER 1001
+USER root
 
 WORKDIR /app
 
-COPY --from=BUILD_IMAGE /usr/src/app/dist /usr/share/nginx/html
+COPY --from=BUILD_IMAGE /usr/src/app/dist /usr/share/amq-spp/dist
+COPY --from=BUILD_IMAGE /usr/src/app/.env /usr/share/amq-spp/.env
+COPY --from=BUILD_IMAGE /usr/src/app/server /usr/share/amq-spp/server
+
+WORKDIR /usr/share/amq-spp
+
+RUN npm install connect
+RUN npm install cors
+RUN npm install express
+RUN npm install express-openapi-validator
+RUN npm install swagger-routes-express
+RUN npm install typescript
+RUN npm install validator
+RUN npm install yamljs
+RUN npm install base-64
+RUN npm install jsonwebtoken
+RUN npm install dotenv
```

Comment:
just a small suggestion here. If I understand correctly, the `RUN` creates new layers. So can it be possible to use `&&` and combine the commands?

Reply:
Thanks Ajay, didn't know that. I think I can do it using && or if I can use a package.json ?

Reply:
yea I guess we can use it both ways 😄 . either in the docker file or as a command in package.json like `install-deps`

Reply:
@AjayJagan I found a easy way to do it. i.e. npm can install multiple package in one command, for example
npm install package1 package 2...